fastseq/optimizer/fairseq/beam_search_optimizer.py

```diff
@@ -17,8 +18,19 @@
 from fairseq.sequence_generator import SequenceGenerator
 from fastseq.ops.ngram_repeat_block import NGramRepeatBlock
 from fastseq.utils.api_decorator import replace
+from fastseq import config
+from fastseq.logging import get_logger
 
-@replace(BeamSearch)
+logger = get_logger(__name__, logging.INFO)
+
+
+#Efficient-Lossless Attention
+use_el_attn = config.USE_EL_ATTN == '1'
+if use_el_attn:
+    logger.info(f"Using Efficient-Lossless Attention optimization")
+
+
+@replace(BeamSearch, True)
```

Comment:
It is not necessary here as 'True' is the default value. Remove it to be concise?

Reply:
done